index.html

```diff
@@ -1304,7 +1374,7 @@ <h2>The <dfn>NDEFRecordType</dfn> enum</h2>
       <td><dfn>"json"</dfn></td>
       <td><a>JSON MIME type</a></td>
       <td>
-        <code>null</code> or <code>DOMString</code> or <code>Number</code> or
+        `null` or <code>DOMString</code> or <code>Number</code> or
```

Comment:
Shouldn't these be {{DOMString}}, {{Number}} or similar?

Reply:
Yes, I'm going to make a unified improvement in a follow-up.

Reply:
There is no such thing as Number, I think? 🤔

Reply:
Right, it should be `UnrestrictedDouble`, I will fix it later. :)